### ISSUE TITLE
Question : Flowfile generation from tcp content

### ISSUE THREAD

**Issue by ddewaele** [apsaltis/nifi-gettcp-bundle#2]
@apsaltis : I was wondering how you see this processor evolve and what kind of FlowFiles should come out of it....

If I understand correctly the goal is to read from a TCP socket using a pre-defined buffer, and send that buffered content as individual flowfiles for further processing ?

Would it also make sense to offer an option to only generate a flowfile after a certain bytesequence occurs (a believe ListenTCP also has an option to specify a flowfile delimiter) ?

I believe this would make upstream processing of msgs a lot easier in the Nifi flow

**Comment by apsaltis** [comment]
@ddewaele You are correct in your understanding and I can totally see the use case for emitting into a flow file the content consumed upto a certain bytesequence. My initial use case did not have this requirement, but I think it makes total sense.

**Comment by ddewaele** [comment]
An alternative would be to use GetTCP + mergeContent (to batch flowfiles) + splitContent (to seperate on bytesequence) processor sequence.

**Comment by apsaltis** [comment]
I can see that working, however, the GetTCP processor may still need to know when a message ends
